decidim-core/lib/tasks/procfile.rake

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require "thor"
+
+namespace :decidim do
+  namespace :procfile do
+    desc "Generates a dummy app for testing in external installations"
+    task :install do
+      actions :create_file, "Procfile.dev", <<~RUBY
+        web: bin/rails server -b 0.0.0.0 -p 3000
+        shakapacker: bin/webpacker-dev-server
+      RUBY
+
+      actions :create_file, "bin/dev", %{#!/usr/bin/env bash
+
+runner=foreman
+
+if command -v overmind &> /dev/null
```

Comment:
```suggestion
if gem list overmind -i --silent
```

Reply:
There is no Gem named overmind ... https://rubygems.org/search?query=overmind

The Overmind library is based on Go. I will keep only foreman.